test/vtcomposite-localize-language.test.js

```diff
@@ -152,67 +151,65 @@ test('[localize] success - feature with specified language in name_{language} pr
 test('[localize] success - feature with specified language in _mbx_name_{language} property', (assert) => {
   const initialBuffer = mvtFixtures.get('063').buffer;
   const initialProperties = {
-    'name': 'Germany',
-    'name_en': 'Germany',
-    'name_fr': 'Allemagne',
-    '_mbx_name_fr': 'La Allemagne',
-    '_mbx_name_de': 'Deutschland',
-    '_mbx_other': 'Alemania'
+    name: 'Germany',
+    name_en: 'Germany',
+    name_fr: 'Allemagne',
+    _mbx_name_fr: 'La Allemagne',
+    _mbx_name_de: 'Deutschland',
+    _mbx_other: 'Alemania'
   };
   const localizedProperties = {
-    'name': 'Deutschland',
-    'name_local': 'Germany',
-    'name_en': 'Germany',
-    'name_fr': 'Allemagne',
+    name: 'Deutschland',
+    name_local: 'Germany',
+    _mbx_other: 'Alemania',
   };
   const initialOutputInfo = vtinfo(initialBuffer);
   const feature = initialOutputInfo.layers.bottom.feature(0);
   assert.deepEqual(feature.properties, initialProperties, 'expected initial properties');
 
   const params = {
     buffer: initialBuffer,
-    language: 'de'
+    languages: ['de']
   };
 
   localize(params, (err, vtBuffer) => {
     assert.notOk(err);
     const outputInfo = vtinfo(vtBuffer);
     const localizedFeature = outputInfo.layers.bottom.feature(0);
-    assert.deepEqual(localizedFeature.properties, localizedProperties, 'expected name_local, updated name, dropped _mbx properties');
+    assert.deepEqual(localizedFeature.properties, localizedProperties, 'expected name_local, updated name, dropped _mbx_name_* properties');
     assert.end();
   });
 });
 
 test('[localize] success - feature with specified language in both name_{language} and _mbx_name_{language} properties', (assert) => {
   const initialBuffer = mvtFixtures.get('063').buffer;
   const initialProperties = {
-    'name': 'Germany',
-    'name_en': 'Germany',
-    'name_fr': 'Allemagne',
-    '_mbx_name_fr': 'La Allemagne',
-    '_mbx_name_de': 'Deutschland',
-    '_mbx_other': 'Alemania'
+    name: 'Germany',
+    name_en: 'Germany',
+    _mbx_name_fr: 'La Allemagne',
+    name_fr: 'Allemagne',
+    _mbx_name_de: 'Deutschland',
+    _mbx_other: 'Alemania'
   };
   const localizedProperties = {
-    'name': 'Allemagne',
-    'name_local': 'Germany',
-    'name_en': 'Germany',
-    'name_fr': 'Allemagne', // choosing first encountered property in (name_fr, _mbx_name_fr)
+    name: 'Allemagne',  // name_{language} takes precedence over _mbx_name_{language}
+    name_local: 'Germany',
+    _mbx_other: 'Alemania'
```

Comment:
why don't we drop arbitrary `_mbx*` fields anymore? 

Ask because I'd expect _someone_ to add fields to their data expecting us to remove them as "internal-only" fields.

Reply:
I made the params for language, worldview and class more consistent:
- Before: we can specify a prefix for language, but not for class and worldview. It isn't clear why.
- After: we can specify a prefix for language, worldview and class. The prefixes can be different.

But that makes dropping fields confusing: for example, do we drop fields that match any of the 3 prefixes? Why do we want to drop a field that is irrelevant to e.g. worldview only because it has the the same prefix as worldview?

If we want to drop arbitrary `_mbx_*` fields, I suggest having just one prefix param and call it something like `params.hidden_prefix`.

Reply:
> If we want to drop arbitrary _mbx_* fields, I suggest having just one prefix param and call it something like params.hidden_prefix.

Irrespective of dropping all fields or not - I like this idea just to simplify the interface. I don't guess someone would set a different prefix for each kind of localizable field, yea?

Reply:
Changed to one `hidden_prefix` and drops any property that starts with this prefix in 00b939803e3f9647113e01a84a9ebb2d382fcabb.